reddit/uploads/api/views.py

```diff
@@ -0,0 +1,28 @@
+from rest_framework import generics
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from users.models import User
+from serializers import PostSerializer, UserPostsSerializer
+
+
+class UserPostsView(generics.ListAPIView):
```

Comment:
What is the purpose of this view? It would fetch all users with all their posts? Not sure how this is relevant

Reply:
Also it doesn't have any authentication?